### PR TITLE
Submission

### PR DESCRIPTION
**Pull request by jtellis**
Pair Programmers: Jesse and David M.

### CODE REVIEW

File: high_card.rb [hunk 1]
```diff
@@ -1,5 +1,95 @@
...
+# players.push({name: "Dick", rank: 0})
+# players.push({name: "Jane", rank: 0})
+# players.push({name: "Buck", rank: 0})
+
+winner = {
+  name: '',
+  rank: 0
+}
+
+def get_rank(card_value)
```
Comment (wdi-pvd) [inline high_card.rb:36]: Very nice!

File: high_card.rb [hunk 2]
```diff
@@ -1,5 +1,95 @@
...
+    rank = 13
+  end
+  return rank
+end
+
+#
+# Not a terribly elegant solution for Bonus 1 and 2
+#
+
+while replay_hand do  # Loop in case of tie
```
Comment (wdi-pvd) [inline high_card.rb:57]: Great use of while loop and setting the conditional in the right spot. How could you import game config settings into your high card program using a config module?

**Comment by wdi-pvd** [comment]
Great job pair programming. Useful commit messages (could be more detailed - imagine a case where you go home at 5pm, and the other person starts working again at 10pm - they need something detailed and in depth so they can pick right up where you left off and not spend time deciphering code). Nice job switching off commits.

**Comment by tvcrawley** [comment]
✅

**Pull request closed without merging**